backend/forecastbox/models/globe.py

```diff
@@ -10,10 +10,12 @@
 
 from typing import Any
 
+from forecastbox.config import config
+
 from .metadata import ControlMetadata
 from .model import BaseForecastModel
 
-DEFAULT_GLOBAL_INPUT_SOURCE = "opendata"
+DEFAULT_GLOBAL_INPUT_SOURCE = config.product.default_input_source or 'opendata'
```

Comment:
we dont need the `or` anymore, do we? Since the config field is non-nullablle and has a default